UIKitHomework/UIKitHomework/NewLabelViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ViewController.swift
+//  UIKitHomework
+//
+//  Created by Polina on 21.06.2021.
+//
+
+import UIKit
+
+final class NewLabelViewController: UIViewController {
+    
+    var yourLabel = UILabel()
```

Comment:
```suggestion
   private var yourLabel = UILabel()
```
вроде нигде не используется вне скоупа